### ISSUE TITLE
TypeError argument str not PosixPath

### ISSUE THREAD

**Issue by akshit-sharma** [BlankSpruce/gersemi#3]
I installed a fresh library through ```pip3 install --user gersemi```.
After installing I tried running ```gersemi .``` in the root of project directory.
But it game me the following error:
```
Traceback (most recent call last):
  File "/home/akshitsharma/.local/bin/gersemi", line 8, in <module>
    sys.exit(main())
  File "/home/akshitsharma/.local/lib/python3.6/site-packages/gersemi/__main__.py", line 163, in main
    sys.exit(run(mode, configuration, args.sources))
  File "/home/akshitsharma/.local/lib/python3.6/site-packages/gersemi/runner.py", line 151, in run
    with create_cache() as cache, create_pool(Path("-") in requested_files) as pool:
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/akshitsharma/.local/lib/python3.6/site-packages/gersemi/cache.py", line 97, in create_cache
    with database_cursor(cache_path()) as cursor:
  File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
    return next(self.gen)
  File "/home/akshitsharma/.local/lib/python3.6/site-packages/gersemi/cache.py", line 38, in database_cursor
    connection = sqlite3.connect(path, detect_types=sqlite3.PARSE_DECLTYPES)
TypeError: argument 1 must be str, not PosixPath
```

I am not sure what I am doing wrong or how to run it.
I also tried running ```gersemi $PWD``` but the same error pops up.

Here's the output from ```gersemi --version```
```
gersemi 0.6.0
lark 0.9.0
Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
[GCC 8.4.0]
```

**Comment by BlankSpruce** [comment]
I can confirm it's a bug that happens with Python 3.6. As a workaround it's not an issue anymore if you upgrade to Python 3.7+.

I'll try to fix this soon.

**Comment by BlankSpruce** [comment]
Issue is fixed in release `0.6.1` so please update gersemi to the latest version.

For reference here's how to reproduce the issue on `0.6.0` with this `Dockerfile`:
```Dockerfile
FROM ubuntu:18.04

RUN \
        apt-get update -y && \
        apt-get install python3 python3-pip -y && \
        mkdir -p ~/foobar

RUN \
        pip3 install --user gersemi==0.6.0 && \
        cd ~/foobar && \
        python3 -m gersemi .
```
